Assets/GameAssets/Player/PlayerScripts/PlayerInteraction.cs

```diff
@@ -62,20 +73,29 @@ private void SetRaycast()
 
     /// <summary>
     /// checks if gameObject can be interacted with
-    ///     If so, calls GiveInteractableFeedback, which should give player feedback that the object is interactable
+    ///     If so, makes interactable text appear in HUD.  Otherwise disables text
     /// </summary>
     /// <param name="gameObject">GameObject that can potentially be interacted with</param>
     private void GiveInteractableFeedback(GameObject gameObject)
     {
         if (gameObject.TryGetComponent(out IPlayerInteractable interactable))
         {
-            //REPLACE THIS WHEN WE MAKE UI FOR INTERACTION
-            Debug.Log("Press E to Interact");
+            if (!_interactUI.activeInHierarchy)
+            {
+                _interactUI.SetActive(true);
+            }
+        }
+        else
+        {
+            if (_interactUI.activeInHierarchy)
```

Comment:
Why are the else and if separate? Why not just do an else if. 
Or could this be simplified down into not even needing it? Perhaps we could just say "_interactUI.SetActive( (the get component && the !_interactUI.activeInHierarchy)" Just spitballing here

Reply:
I made the change to reflect else if.  The purpose of the nested if statement for each case is to not unnecessarily call SetActive when the object is already in the state I want it to be in, hence why the 2nd option doesn't work.  See the forwarded message in the PR thread